src/zkbob/ZkBobPool.sol

```diff
@@ -42,6 +44,11 @@ abstract contract ZkBobPool is IZkBobPool, EIP1967Admin, Ownable, Parameters, Zk
     address public immutable token;
     IZkBobDirectDepositQueue public immutable direct_deposit_queue;
 
+    uint256[3] private __deprecatedGap;
```

Comment:
Why is it necessary to reserve three storage slots? ZkBobAccounting used only slot0 and slot1 in the past as far as I see: 
https://github.com/zkBob/zkbob-contracts/blob/e9fca9996535e80297e3e24c248aaf5ac4b1e409/src/zkbob/utils/ZkBobAccounting.sol#L113-L114

Is it the 3rd slot to keep the Token Seller contract address?